docs/decisions/0001-use-npm-to-manage-npm-dependencies-for-deno-projects.md

```diff
@@ -1,3 +1,7 @@
+---
+title: 0001 - Use `npm` to manage NPM dependencies for Deno projects
```

Comment:
nit: I think the title looks better when render without the backticks since we don't format it as a codeblock in the sidenav:

```suggestion
title: 0001 - Use npm to manage NPM dependencies for Deno projects
```